js/index.js

```diff
@@ -3,29 +3,47 @@
 function updateSubtotal(product) {
   console.log('Calculating subtotal, yey!');
 
-  //... your code goes here
+  const price = product.querySelector('.price span');
+  const quantity = product.querySelector('.quantity input').value;
+  const subt = price.innerHTML * quantity
```

Comment:
```suggestion
  const subt = price * quantity;
```
Cuando ya tienes el innerText ya no te hace falta sacarlo aqui, y tambien te digo intenta evitar el innerHTML, solo para cuando sea inevitable usarlo, si no caca de la buena xD